crates/ruff/src/checkers/logical_lines.rs

```diff
@@ -250,5 +250,20 @@ f()"#
             "f()",
         ];
         assert_eq!(actual, expected);
+
+        let contents = r#"
+if False:
+
+    print()
+"#
+        .trim();
+        let lxr: Vec<LexResult> = lexer::lex(contents, Mode::Module).collect();
+        let locator = Locator::new(contents);
+        let actual: Vec<String> = LogicalLines::from_tokens(&lxr, &locator)
+            .into_iter()
+            .map(|line| line.text_trimmed().to_string())
+            .collect();
+        let expected = vec!["if False:", "print()", ""];
+        assert_eq!(actual, expected);
```

Comment:
Nit: Can we split this into its own test. Giving tests an explicit name help future us because it gives another hint on what the test is asserting and also allows to run the test on its own.

Reply:
Gonna do this as its own PR.